docs/components/plugins/CodeBlocksTable.tsx

```diff
@@ -34,12 +37,16 @@ export function CodeBlocksTable({ children, tabs, connected = true, ...rest }: P
       return MDX_CLASS_NAME_TO_TAB_NAME[className] || className.replace('language-', '');
     });
 
+  console.warn(codeBlocks);
```

Comment:
Out of curiosity, is this console statement intentional?

Reply:
Ohh, we need to fix that, nice catch!